sentry/src/main/java/io/sentry/Scopes.java

```diff
@@ -677,7 +679,7 @@ public void configureScope(
               "Instance is disabled and this 'configureScope' call is a no-op.");
     } else {
       try {
-        callback.run(combinedScope.getSpecificScope(scopeType));
+        callback.run(combinedScope.getSpecificScope(scopeType, true));
```

Comment:
Is there a specific reason for the new boolean param in `getSpecificScope`? Couldn't we just use `combinedScope` here? Or do we expect there to be more callbacks that would use this in the future?

Reply:
If we pass in `ScopeType.COMBINED` here, we'd always use combined scope. If a user passes a specific scope into `configureScope` that scope should be used. We only want to change the default scope type for the callbacks not always use combined scope.